BasedHardware/Omi
# Added Friend Speaker Support and Buzzer #567 #573

#567 

How to test:
In the firmware_v1.0 file,
- Flash the zephyr.uf2 file in the file "build_speaker". 
-Then use the script "discover_devices.py" to find the device id of your Omi device. This is to avoid cases where there are multiple friends connected at once. If there is more than one friend then just test all of them.

In the script play_sound_on_friend.py, (in firmware_v1.0)
- On line 20 add the device id of your friend. This is to avoid cases where there is more than one friend nearby.
- On line 21 add your Deepgram API key.

Then save and run the script by typing a small string. For example, one line could be

python3 play_sound_on_friend.py "hello"

Which should get the speaker to say hello. Make sure your device is connected by USB or you have connected a 3.3V pin to the speaker.


#573  (For now it is disabled but since this uses very similar code). 

A sound should play automatically when you compile and run the zephyr.uf2 file. Of course this file can be modified to play any haptic sound provided you enter the appropriate lookup table.
